src/utils_GUI/GUI_utils.jl

```diff
@@ -398,17 +422,63 @@ function initialize_available_data!(gui)
 end
 
 """
-    get_JuMP_axes(var::JuMP.Containers.DenseAxisArray, i_T::Int)
+    get_investment_times(gui::GUI, max_inst::Float64)
 
-Get arrays of indices of a model variable `var` not being the time dimension (located at index `i_T`)
+Calculate when investments has taken place and store the information. An investement is
+assumed to have taken place if any `investment_indicators` are larger than get_var(gui,:tol)
+relative to `max_inst`.
 """
-function get_JuMP_axes(var::JuMP.Containers.DenseAxisArray, i_T::Int)
-    return axes(var)[vcat(1:(i_T - 1), (i_T + 1):end)]
+function get_investment_times(gui::GUI, max_inst::Float64)
```

Comment:
9 indentation levels is a bit too much?

Reply:
Maybe, but it is a result of using the break setup here. Feel free to suggest alternatives :)